lib/BloqadeQMC/src/ising/Rydberg.jl

```diff
@@ -1,23 +1,36 @@
 using Base.Iterators
-
+using BloqadeLattices: rydberg_interaction_matrix, AtomList
 
 abstract type AbstractRydberg{O <: AbstractOperatorSampler} <: AbstractLTFIM{O} end
 
-struct Rydberg{O,M <: UpperTriangular{Float64},UΩ <: AbstractVector{Float64}, Uδ <: AbstractVector{Float64}, L <: Lattice} <: AbstractRydberg{O}
+# struct Rydberg{O,M <: UpperTriangular{Float64},UΩ <: AbstractVector{Float64}, Uδ <: AbstractVector{Float64}, L <: Lattice} <: AbstractRydberg{O}
```

Comment:
You don't need to comment out code. We can always go back in the code history to check out anything that gets erased :)

Reply:
Ok, I'll erase it then! :)